packages/stark-ui/tsconfig-build.json

```diff
@@ -3,7 +3,13 @@
   "compilerOptions": {
     "baseUrl": ".",
     "rootDir": ".",
-    "typeRoots": ["./node_modules/@types", "../stark-build/typings", "./typings"],
+    "typeRoots": [
+      "./node_modules/@types",
+      "./node_modules/@nationalbankbelgium/stark-testing/node_modules/@types",
```

Comment:
Now, thanks to the changes done in https://github.com/NationalBankBelgium/stark/pull/1207, there is no need to add ` "./node_modules/@nationalbankbelgium/stark-testing/node_modules/@types",` to the `typeRoots`.